apps/electron-app/src/main/store/user-profile-store.ts

```
@@ -0,0 +1,2174 @@
+    // Clear session data
+    profileSession.clearStorageData();
+    profileSession.clearCache();
+
+    // Remove from maps
+    set(state => {
+      const newSessions = new Map(state.profileSessions);
+      newSessions.delete(profileId);
+
+      const profile = state.profiles.get(profileId);
+      if (profile) {
+        delete profile.session;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)